packages/sources/src/components/form/models/form-group-static.model.ts

```diff
@@ -0,0 +1,9 @@
+import { Icon } from '../../..';
+
+import { FormGroupBase } from './form-group.base-model';
+
+export interface FormGroupStatic<TemplateFnReturnType> extends FormGroupBase<TemplateFnReturnType> {
+  group: 'static',
+  edit: boolean,
+  value?: string
```

Comment:
Zou `value` niet gewoon verplicht moeten zijn? Zonder value is er niets te tonen bij een static.